core/src/labels/label.h

```diff
@@ -186,6 +191,8 @@ class Label {
 
     int m_anchorIndex;
 
+    uint32_t m_selectionColor;
```

Comment:
You can use the selection color from Sprite/TextLabel `m_vertexAttrib.selectionColor`